diffs/heartbeat.diff

```diff
@@ -0,0 +1,1933 @@
+@@ -727,16 +731,18 @@
+               </div>
+              </div>
+             </div>
+-            <div class="warning admon">
+-             <div class="icon">
+-              <img alt="Warning" src="images/icons/warning.png"/>
+-             </div>
+-             <div class="admon_content">
+-              <p>
++            <p>
++             <span class="Admonishment">
++              [
++              <span class="Admonishment-version u-mono u-strikethrough">
++               5.0.0
++              </span>
++              ]
++              <span class="Admonishment-detail">
+                Deprecated in 5.0.0.
+-              </p>
+-             </div>
+-            </div>
++              </span>
++             </span>
++            </p>
+             <p>
+              The Redis database number where the topology information is stored. The default is 1.
+             </p>
+@@ -746,7 +752,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_worker_literal_4">
++                <a id="_worker_4">
+                 </a>
+                 <code class="literal">
+                  worker
+@@ -771,7 +777,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_loadbalance_literal">
++                <a id="_loadbalance">
+                 </a>
+                 <code class="literal">
+                  loadbalance
+@@ -792,7 +798,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_timeout_literal_4">
++                <a id="_timeout_4">
+                 </a>
+                 <code class="literal">
+                  timeout
+@@ -813,7 +819,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_backoff_init_literal_3">
++                <a id="_backoff_init_3">
+                 </a>
+                 <code class="literal">
+                  backoff.init
+@@ -842,7 +848,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_backoff_max_literal_3">
++                <a id="_backoff_max_3">
+                 </a>
+                 <code class="literal">
+                  backoff.max
+@@ -863,7 +869,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_max_retries_literal_4">
++                <a id="_max_retries_4">
+                 </a>
+                 <code class="literal">
+                  max_retries
+@@ -894,7 +900,7 @@
+              <div>
+               <div>
+                <h4 class="title">
+-                <a id="_literal_bulk_max_size_literal_4">
++                <a id="_bulk_max_size_4">
+                 </a>
+                 <code class="literal">
+                  bulk_max_size
+@@ -932,7 +938,7 @@
+              <div>
+        
```

Comment:
Fixed in https://github.com/elastic/beats/commit/ebb0e742cf0953397da473e5d9a931e1b55a2a4c